content/docs/authorization/uninstallation.md

```diff
@@ -10,9 +10,10 @@ This section outlines the uninstallation steps for Container Storage Modules (CS
 
 ## Uninstalling the RPM
 
-To uninstall the rpm package on the system, run the below command:
+To uninstall the rpm package on the system, you must first uninstall the K3s SELinux policy. To uninstall, run the below commands in the following order:
```

Comment:
Please mention that k3s-selinux uninstall is only required is SELinux is enabled. Also change 'policy' to package.

Reply:
Done please review